### PR TITLE
[PLAT-1678] Change to use new objdump command 2X

### PR DESCRIPTION
**Pull request by Pezzah**
## Goal

Use faster version of objdump command

Requires https://github.com/bugsnag/bugsnag-event-worker/pull/1061 to parse the new file upload

## Review

For the submitter, initial self-review:

- [ ] Commented on code changes inline explain the reasoning behind the approach
- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] A changelog entry was added for the goal of this pull request
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?
- This pull request is ready for:
  - [ ] Initial review of the intended approach, not yet feature complete
  - [ ] Structural review of the classes, functions, and properties modified
  - [ ] Final review

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency across platforms for structures or concepts added or modified
- [ ] Consistency between the changeset and the goal stated above
- [ ] Internal consistency with the rest of the library - is there any overlap between existing interfaces and any which have been added?
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Performance and complexity - are there any cases of unexpected O(n^3) when iterating, recursing, flat mapping, etc?
- [ ] Concurrency concerns - if components are accessed asynchronously, what issues will arise
- [ ] Thoroughness of added tests and any missing edge cases
- [ ] Idiomatic use of the language

### CODE REVIEW

File: src/main/groovy/com/bugsnag/android/gradle/BugsnagUploadNdkTask.groovy [hunk 1]
```diff
@@ -152,7 +152,7 @@ class BugsnagUploadNdkTask extends BugsnagUploadAbstractTask {
                 // Call objdump, redirecting output to the output file
                 project.logger.info("${getLogPrefix()} Calling objdump process")
 
-                ProcessBuilder builder = new ProcessBuilder(objDumpPath.toString(), "--disassemble", "--demangle", "--line-numbers", "--section=.text", sharedObject.toString())
+                ProcessBuilder builder = new ProcessBuilder(objDumpPath.toString(), "-W", "-x", "--section=.debug_line", sharedObject.toString())
```
Comment (kattrali) [inline src/main/groovy/com/bugsnag/android/gradle/BugsnagUploadNdkTask.groovy]: What subset of `-W` is actually necessary (since it wraps [several options](https://sourceware.org/binutils/docs/binutils/objdump.html))? This file size could be reduced if we know specifically which components are required.
